crates/re_types/Cargo.toml

```diff
@@ -102,6 +100,10 @@ re_types_builder.workspace = true
 rayon.workspace = true
 
 
+[package.metadata.cargo-machete]
+ignored = ["rayon", "re_build_tools", "re_types_builder"]
```

Comment:
do we know why machete is struggling with these?

Reply:
`machete` just seems to strugle with `build-dependencies` in general